pkg/driver/camera/camera_linux.go

```diff
@@ -162,7 +164,7 @@ func (c *camera) Open() error {
 	}
 
 	// Buffering should be handled in higher level.
-	err = cam.SetBufferCount(1)
+	err = cam.SetBufferCount(uint32(bufCount))
```

Comment:
Const integer in Go is not typed by default, so
```suggestion
	err = cam.SetBufferCount(bufCount)
```

Reply:
Ah, gotcha. Done!